server/config.go

```diff
@@ -49,8 +50,9 @@ type CachingConfig struct {
 }
 
 type WorkerConfig struct {
-	Workers   int `yaml:"workers"`
-	QueueSize int `yaml:"queue_size"`
+	Workers       int           `yaml:"workers"`
+	QueueSize     int           `yaml:"queue_size"`
+	ClientTimeout time.Duration `yaml:"client_timeout"`
```

Comment:
considering the workers are the ones making the requests, i put the configuration in this block. open to having it in a different location and a different name

Reply:
I think here makes the most sense, followed the the generic "options" config.

What do you think about calling this `request_timeout` or `github_timeout`? I think it's kind of ambiguous what "client" refers to here.

Reply:
keeping here and renaming to `github_timeout`